packaging/package-indexer/index-packages.py

```diff
@@ -27,7 +27,7 @@
 from sys import stdin
 from typing import List
 
-from indexer import Indexer, NightlyDebIndexer, ReleaseDebIndexer
+from indexer import Indexer, NightlyDebIndexer, StableDebIndexer
```

Comment:
Currently `ReleaseRpmIndexer` is not used, nevertheless shouldn't we rename it too?

Reply:
Yeah, probably I should not have added it back then... I think it would be better to remove them altogether. We can add it back if we need them.

Reply:
Removed.